R/get.R

```diff
@@ -112,7 +112,7 @@ get_stats19 = function(year = NULL,
     )
     output_format = "tibble"
   }
-  if (grepl(type, "casualties", ignore.case = TRUE) && output_format %in% c("sf", "ppp")) {
+  if (grepl("casualties", type, ignore.case = TRUE) && output_format %in% c("sf", "ppp")) {
```

Comment:
```suggestion
  if (grepl("cas", type, ignore.case = TRUE) && output_format %in% c("sf", "ppp")) {
```

And save for "veh" and "acc|col" so more things match.